rts/lra/lra-coordinator/pom.xml

```diff
@@ -51,6 +51,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${surefire-plugin.version}</version>
                 <configuration>
+                    <redirectTestOutputToFile>${test.logs.to.file}</redirectTestOutputToFile>
```

Comment:
please, remove this line. The `lra-coordinator` has no tests included under the directory structure and this is not needed.

Reply:
okay